parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
Just checked 2 systems and none of them have a lib64 or lib32 folder in `/usr` , but both have the `/usr/lib/<arch_triplet>` folder